beacon-chain/blockchain/process_block.go

```diff
@@ -571,6 +572,7 @@ func (s *Service) isDataAvailable(ctx context.Context, root [32]byte, signed int
 			s.blobNotifiers.delete(root)
 			return nil
 		case <-ctx.Done():
+			debug.SaveBlockToTemp(signed)
 			return errors.Wrap(ctx.Err(), "context deadline waiting for blob sidecars")
```

Comment:
What's the case for this?

Reply:
When the user toggles `--save-unavailable-block-temp`, that is when one wants to inspect a valid block without any follow-up blobs. We want to check the block's commitments, who the builder is, etc.